R/iconswithtext.R

```diff
@@ -201,14 +202,14 @@ iconsWithText <- function (x,
     # padding format: top right bottom left
     pad.above.left <- pad.above.right <- pad.above.top <- pad.above.bottom <- 0
     pad.below.left <- pad.below.right <- pad.below.top <- pad.below.bottom <- 0
-    if (sum(nchar(text.above), na.rm = TRUE) > 0)
+    if (Sum(nchar(text.above)) > 0)
```

Comment:
Can this be replaced by `any(nzchar(text.above))`?

Reply:
Quite possibly. I'll look into it.

Reply:
Changed. Also removed the use of `flipStatistics::Sum` and replacing with `verbs::Sum`. It seems compatible after checking its use in the Standard R pages.